src/common/log/outputChannelLogFormatter.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+/**
+ * Formatter for the Output channel.
+ */
+
+import {LogHelper} from "./logHelper";
+
+export class OutputChannelLogFormatter {
+
+    public static getFormattedMessage(message: string) {
+        return OutputChannelLogFormatter.getFormattedOutputChannelString(message);
```

Comment:
Why have this additional level of indirection, and not just directly expose `getFormattedOutputChannelString`?